cypress/integration/options/legend.cy.js

```diff
@@ -0,0 +1,12 @@
+/*  TODO:
```

Comment:
Instead of having test files that only have TODOs and no code, maybe there should be a jira epic with issues for each of these TODOS and FIXMEs?

Reply:
Yeah would be good to plan what needs to be tested and refactored in a central place. @janhenrikoverland ?

Reply:
We've discussed this and leaving the comments as is for now makes the most sense. We can start a proper JIRA epic once we pick up the holistic regression test writing again in the future. For now we'll only add continuous feature tests as we go.